examples/Delegates.kt

```diff
@@ -0,0 +1,14 @@
+// Taken from Kotlin 1.7
+// https://kotlinlang.org/docs/whatsnew17.html#underscore-operator-for-type-arguments
+
+interface Bar {
+    fun foo() = "foo"
+}
+
+@JvmInline
+value class BarWrapper(val bar: Bar): Bar by bar
```

Comment:
I believe, the `value` modifier cannot be parsed by `tree-sitter-kotlin` currently (should be easy to fix though).